modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -295,22 +295,25 @@ properties:
           revs_limit:
             type: integer
             description: |
-              Maximum depth to which a document's revision tree can grow. In Sync Gateway 1.5, the `revs_limit` configuration cannot be set lower than 20. Setting the `revs_limit` to a low value (typically below 100) can have adverse effects as described below.
-              
-              The value of `revs_limit` governs the point at which to prune a document's revision tree.
-              
-              Indeed, the process to remove obsolete revisions is called pruning and runs automatically every time a revision is added. Although fundamentally the same, the pruning algorithm works slightly differently between Sync Gateway and Couchbase Lite. On Sync Gateway, the pruning algorithm is applied to the shortest, non-tombstoned branch in the revision tree.
-            
-              If there are conflicting revisions, the document may end up with **disconnected branches** after the pruning process. In the animation below, the document has a conflicting branch (revisions `4'` - `1001'`). When the shortest branch (in this case the conflicting branch) reaches the 1003rd update, it gets is cut off. The revision tree is not in a corrupted state and the logic that chooses the winning revision still applies. But it may make it impossible to do certain merges (n-way merge) to resolve conflicts and will occupy disk space that could have been free-ed if the conflict was resolved early on.<br><br>
-            
+              Maximum depth to which a document's revision tree can grow, it governs the point at which to prune a document's revision tree. Revision pruning is the process to remove obsolete parent revisions and runs every time a revision is added.
+
+              The minimum value is 20.
```

Comment:
```suggestion
              The minimum and default values that this property can take is dependent on the value of the  [conflict-free](https://docs.couchbase.com/sync-gateway/2.6/config-properties.html#databases-foo_db-allow_conflicts) config setting
```

Reply:
- If [allow-conflicts](https://docs.couchbase.com/sync-gateway/2.6/config-properties.html#databases-foo_db-allow_conflicts) property is `true` then the default value is 100 and min value is 20
- If [allow-conflicts](https://docs.couchbase.com/sync-gateway/2.6/config-properties.html#databases-foo_db-allow_conflicts) property is `false` then the default value is 50 and min value is 1